tests/components/wsdot/test_sensor.py

```diff
@@ -16,56 +13,46 @@
     RESOURCE,
     SCAN_INTERVAL,
 )
-from homeassistant.setup import setup_component
+from homeassistant.setup import async_setup_component
+
+from tests.common import load_fixture
+
+config = {
+    CONF_API_KEY: "foo",
+    SCAN_INTERVAL: timedelta(seconds=120),
+    CONF_TRAVEL_TIMES: [{CONF_ID: 96, CONF_NAME: "I90 EB"}],
+}
+
 
-from tests.common import get_test_home_assistant, load_fixture
+async def test_setup_with_config(hass):
+    """Test the platform setup with configuration."""
+    assert await async_setup_component(hass, "sensor", {"wsdot": config})
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)